lib/media/drm_engine.js

```diff
@@ -1106,9 +1106,19 @@ shaka.media.DrmEngine.prototype.sendLicenseRequest_ = function(event) {
  * @private
  */
 shaka.media.DrmEngine.prototype.unpackPlayReadyRequest_ = function(request) {
-  // The PlayReady license message as it comes from the CDM can't be directly
-  // delivered to a license server.  Other CDMs do not seem to need this kind
-  // of special handling.
+  // PlayReady CDMs in some clients (e.g. IE11, Edge) wrap the license message
+  // in UTF-16 encoded XML which can't be directly delivered to a license
+  // server. However, not all clients exhibit this behaviour. The Tizen
+  // PlayReady CDM message is UTF-8 encoded and can be passed to the license
+  // server as-is. Other CDMs do not seem to need this kind of special
+  // handling.
+  var xml = String.fromCharCode.apply(null, request.body);
```

Comment:
request.body is typed as an ArrayBuffer, so you will need to wrap that in "new Uint8Array".

Here's a demo of fromCharCode applied to both a Uint8Array and an ArrayBuffer:

u = new Uint8Array([61, 62, 63, 64, 65]);

String.fromCharCode.apply(null, u);
"=>?@A"

String.fromCharCode.apply(null, u.buffer);
""

If request.body is a Uint8Array at runtime, wrapping it again will not cause a problem.